src/subsets/Subsets.jsx

```diff
@@ -0,0 +1,263 @@
+    let msaMds = stateToMsas[subsetYear][this.state.selectState.value];
+    let msaMdsOptions = msaMds
+      ? msaMds.map(msaMd => {
+          return { value: msaMd.id, label:  msaMd.id+" - "+msaMd.name };
+        })
+      : [{ id: "", value: "" }];
+
+    return (
+      <Select
+        isDisabled={this.state.selectState.value ? false : true}
+        onChange={this.handleMsaMdsUpdate}
+        placeholder="Select MSA/MD..."
+        searchable={true}
+        autoFocus
+        openOnFocus
+        simpleValue
+        options={msaMdsOptions}
+        value={this.state.selectMsaMds}
+      />
+    );
+  }
+
+  loadVarSelect(varOption, varListName) {
+    let varList = variables[varOption][varListName];
+    let varOptions = varList
+      ? varList.map(variableOption => {
+          return { value: variableOption.id, label: variableOption.name };
+        })
+      : [{ id: "", value: "" }];
+    return (
+      <Select
+        isDisabled={this.state.selectState.value ? false : true}
+        onChange={
+          varOption === "1"
+            ? this.handleActionTakenUpdate
+            : this.handleRaceUpdate
+        }
+        placeholder={"Select variable" + varOption + "..."}
+        searchable={true}
+        autoFocus
+        openOnFocus
+        simpleValue
+        options={varOptions}
+        value={
+          varOption === "1"
+            ? this.state.selectActionTaken
+            : this.state.selectRace
+        }
+      />
+    );
+  }
+
+  onStateChange(selectedOption) {
+    return this.setState({
+      selectState: selectedOption,
+      selectMsaMds: "",
+      selectActionTaken: "",
+      selectRace: ""
+    });
+  }
+  onMsaMdsChange(selectedOption) {
+    return this.setState({
+      selectMsaMds: selectedOption
+    });
+  }
+
+  handleActionTakenUpdate(selectedOption) {
+    return this.setState({
+      selectActionTaken: selectedOption.length?"":selectedOption
+    });
+  }
+
+  handleRaceUpdate(selectedOption) {
+    return this.setState({
+      selectRace: selectedOption.length?"":selectedOption
+    });
+  }
+
+  render() {
+    const {location}  = this.props;
+    const subsetYear = location?location.pathname.split("/")[2]:"NA";
+
+    return (
+      <div className="Subsets">
+        <div className="intro">
+          <Header type={1} headingText="Subsets of HMDA data">
+            <p className="lead">
+              Customize your analysis of HMDA data. Create subsets of data here,
+              using pre-selected filters that allow you to compare queries. For
+              questions/suggestions, contact hmdafeedback@cfpb.gov.
+            </p>
+          </Header>
+        </div>
+        <table>
+          <thead />
+          <tbody>
+            <tr>
+              <th width="50%">Select a State (or Nationwide): </th>
+              <th width="50%">Choose an available MSA/MD:</th>
+            </tr>
+            <tr>
+              <td width="50%">
+                <Select
```

Comment:
Since selecting states and MSAs are somewhat independent, perhaps it would be worth only having one select with the full list of states and msas together. MSAs could be bucketed into states and searching would involve either typing the msa number directly or first typing the state to filter down to just its MSAs and browse from there.

This might be a better way to allow selecting state/MSA individually.